addons/delivery/views/report_deliveryslip.xml

```diff
@@ -21,6 +21,14 @@
         <xpath expr="//table[@name='stock_move_line_table']/thead/tr" position="inside">
             <th t-if="has_hs_code"><strong>HS Code</strong></th>
         </xpath>
+        <xpath expr="//table[@name='stock_move_table']/thead/tr" position="inside">
```

Comment:
Instead of adding a new one, isn't the previous `th` xpath who should be changed to have a similar path than the one you use for the `td` ?
```xml
<xpath expr="//tr[last()]" position="after">
```